pgml-extension/src/bindings/transformers.py

```diff
@@ -97,10 +119,14 @@ def __init__(self, model_name, **task):
 
     def __call__(self, inputs, **kwargs):
         outputs = []
+        pipe = pipeline(
+            self.task,
+            model=self.model,
+            tokenizer=self.tokenizer,
+            **kwargs
+        )
         for input in inputs:
-            tokens = self.tokenizer(input, return_tensors="pt").to(self.model.device).input_ids
-            token_ids = self.model.generate(input_ids=tokens, **kwargs)[0]
-            outputs.append(self.tokenizer.decode(token_ids))
+            outputs.append(pipe(input)[0]['generated_text'])
```

Comment:
The result of the pipe, `generated_text`, doesn't seem to be 100% standardized across models. I think this will cause an error for those cases that return a string instead of a dict.

Reply:
Do you have an example model that is returning something different so I can make this more robust?

Reply:
I think any transformer that doesn't use the `text-generation` task will have different output format, like `translation`, `summarization` or `text-classification`. I'm not sure if those tasks also have specific pipeline output formats. That'd be interesting to also handle more cleanly, but changing the output format would also be a breaking API change for anyone currently using it as is.